### PR TITLE
ENH: Mini labels

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: sampledb/generate_qr.py [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
-intercol = 0.125
+intercol = 0.6
 interrow = 0
 rbord = 0.0625
 lbord = 0.0625
 tbord = 0.125
 bbord = 0.125
 width = (8.5 - lmar - rmar - (cols - 1) * intercol) / cols - lbord - rbord
-height = (11 - tmar - bmar - (rows - 1) * interrow) / rows - tbord - bbord
+height = .8 *(11 - tmar - bmar - (rows - 1) * interrow) / rows - tbord - bbord
```
Comment (justcalamari) [inline sampledb/generate_qr.py]: Missing whitespace. Should be `.8 * (11 ...`

File: sampledb/templates/qr_template.tex [hunk 2]
```diff
@@ -19,13 +19,12 @@
...
 %\LabelGridtrue
-\numberoflabels=\LabelCols
+\numberoflabels=1
 \begin{document}
 \begin{labels}
 \BLOCK{for qr, uid in qrs}
-    \centering
-        \includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\\
-    \captionof*{figure}{\VAR{uid}}
+        \hfill\includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\hfill\includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\hfill\includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\hfill\\
```
Comment (justcalamari) [inline sampledb/templates/qr_template.tex]: There is probably an extra tab here. We probably want the same indentation as the next line.

**Comment by CJ-Wright** [comment]
This still needs a news entry

**Comment by CJ-Wright** [comment]
@justcalamari please merge when this when you are satisfied and it passes

**Comment by CJ-Wright** [comment]
Thank you very much!

**Pull request merged**